src/main/java/org/apache/commons/imaging/color/ColorConversions.java

```diff
@@ -49,9 +43,9 @@ public static ColorCieLab convertXYZtoCIELab(final double X, final double Y, fin
         double var_Z = Z / REF_Z; // REF_Z = 108.883
 
         // Pivot XÝZ:
-        var_X = pivotXYZ(var_X);
-        var_Y = pivotXYZ(var_Y);
-        var_Z = pivotXYZ(var_Z);
+        var_X = pivotProcessor.pivotXYZ(var_X);
+        var_Y = pivotProcessor.pivotXYZ(var_Y);
+        var_Z = pivotProcessor.pivotXYZ(var_Z);
```

Comment:
Variables in Imaging may have weird names but these many times match what is described on papers and specifications. Refactorings like these are not really helpful.